.github/workflows/choreo.yml

```diff
@@ -71,7 +75,7 @@ jobs:
         if: startsWith(matrix.os, 'macOS')
 
       - name: Build package
-        run: npm run tauri build ${{ matrix.tauri-build-flags }}
+        run: pnpm tauri build ${{ matrix.tauri-build-flags }}
```

Comment:
Why does lint-format.yml use `pnpm run` while this doesn't?

Reply:
'Twas an attempt at debugging Mac and windows arm failures